pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/model/XPathSuggestions.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.util.fxdesigner.model;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.List;
+
+
+public class XPathSuggestions {
+    private List<String> xPathSuggestions = new ArrayList<>();
+    String packagename;
```

Comment:
if this is intented to be package-private, document it. If it's not, please change it to proper visibility (private?)